extensions/amp-story/1.0/test/validator-amp-story-live-story.out

```diff
@@ -0,0 +1,57 @@
+|    you may not use this file except in compliance with the License.
+|    You may obtain a copy of the License at
+|        http://www.apache.org/licenses/LICENSE-2.0
+|    Unless required by applicable law or agreed to in writing, software
+|    distributed under the License is distributed on an "AS-IS" BASIS,
+|    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+|    See the License for the specific language governing permissions and
+|    limitations under the license.
+|  -->
+|  <!--
+|    Test Description:
+|    Tests for the live story functionality inside amp-story.
+|  -->
+|  <!doctype html>
+|  <html amp lang="en">
+|    <head>
+|      <meta charset="utf-8">
+|      <script async src="https://cdn.ampproject.org/v0.js"></script>
+|      <script async custom-element="amp-story"
+|          src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+|      <script async custom-element="amp-live-list"
+|          src="https://cdn.ampproject.org/v0/amp-live-list-0.1.js"></script>
+|      <title>amp-story live story diff</title>
+|      <meta name="viewport"
+|            content="width=device-width,minimum-scale=1,initial-scale=1">
+|      <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+|      <link rel="canonical" href="index.html">
+|      <style amp-custom>
+|        amp-story-page {
+|          background: black;
+|          color: white;
+|          font-family: sans-serif;
+|        }
+|      </style>
+|    </head>
+|
+|    <body>
+|      <amp-story standalone publisher="AMP Team" title="Visual Diff Test"
+|          publisher-logo-src="/examples/visual-tests/photos/120.png"
+|          poster-portrait-src="/examples/visual-tests/picsum.photos/image981_900x1600.jpg"
+|          poster-landscape-src="/examples/visual-tests/picsum.photos/image981_1600x900.jpg"
+|          poster-square-src="/examples/visual-tests/picsum.photos/image981_1600x1600.jpg"
+|          id="story1" live-story disabled data-poll-interval="1500">
+|
+|        <amp-story-page id="cover" data-sort-time="1552330790">
+|          <amp-story-grid-layer template="vertical">
+|            <p>Page one of one</p>
+|          </amp-story-grid-layer>
+|        </amp-story-page>
+|      </amp-story>
+|    </body>
+|
+|  </html>
```

Comment:
nit: newline, as usual

Sorry, I don't know if this has already been talked about, but @Gregable should we either (a) fix the tooling to output these files with newlines at the end, or (b) ignore the "newline at the end of file" rule for the validator `.out` files?